packages/js/src/base-module.ts

```diff
@@ -22,22 +21,22 @@ export class BaseModule {
 
   constructor() {
     this._emitter = NovuEventEmitter.getInstance();
-    this._apiService = ApiServiceSingleton.getInstance();
     this._inboxService = InboxServiceSingleton.getInstance();
-    this._emitter.on('session.initialize.success', ({ result }) => {
-      this.onSessionSuccess(result);
-      this.#callsQueue.forEach(async ({ fn, resolve }) => {
-        resolve(await fn());
-      });
-      this.#callsQueue = [];
-    });
-    this._emitter.on('session.initialize.error', ({ error }) => {
```

Comment:
the `.success` and `.error` events were consolidated into one `.resolved` that is called with `{ data, error }` props